pallets/pools/src/benchmarking.rs

```diff
@@ -49,76 +49,76 @@ benchmarks! {
 	}
 
-	create {
-		let n in 1..T::MaxTranches::get();
-		let caller: T::AccountId = create_admin::<T>(0);
-		let tranches = build_bench_input_tranches::<T>(n);
-		let origin = RawOrigin::Signed(caller.clone());
-		prepare_asset_registry::<T>();
-	}: create(origin, caller, POOL, tranches.clone(), CurrencyId::AUSD, MAX_RESERVE, None)
-	verify {
-		let pool = get_pool::<T>();
-		assert_input_tranches_match::<T>(pool.tranches.residual_top_slice(), &tranches);
-		assert_eq!(pool.reserve.available, Zero::zero());
-		assert_eq!(pool.reserve.total, Zero::zero());
-		assert_eq!(pool.parameters.min_epoch_time, T::DefaultMinEpochTime::get());
-		assert_eq!(pool.parameters.max_nav_age, T::DefaultMaxNAVAge::get());
-		assert_eq!(pool.metadata, None);
-	}
-
-	update_no_execution {
-		let admin: T::AccountId = create_admin::<T>(0);
-		let n in 1..T::MaxTranches::get();
-		let tranches = build_update_tranches::<T>(n);
-		prepare_asset_registry::<T>();
-		create_pool::<T>(n, admin.clone())?;
-		let pool = get_pool::<T>();
-		let default_min_epoch_time = pool.parameters.min_epoch_time;
-		let default_max_nav_age = pool.parameters.max_nav_age;
-
-		// Submit redemption order so the update isn't executed
-		let amount = MAX_RESERVE / 2;
-		let investor = create_investor::<T>(0, TRANCHE)?;
-		let locator = get_tranche_id::<T>(TRANCHE);
-		Pallet::<T>::update_redeem_order(RawOrigin::Signed(investor.clone()).into(), POOL, tranche_location::<T>(TRANCHE), amount)?;
-
-		let changes = PoolChanges {
-			tranches: Change::NoChange,
-			min_epoch_time: Change::NewValue(SECS_PER_DAY),
-			max_nav_age: Change::NewValue(SECS_PER_HOUR),
-			tranche_metadata: Change::NoChange,
-		};
-	}: update(RawOrigin::Signed(admin), POOL, changes.clone())
-	verify {
-		// Should be the old values
-		let pool = get_pool::<T>();
-		assert_eq!(pool.parameters.min_epoch_time, default_min_epoch_time);
-		assert_eq!(pool.parameters.max_nav_age, default_max_nav_age);
-
-		let actual_update = get_scheduled_update::<T>();
-		assert_eq!(actual_update.changes, changes);
-	}
-
-	update_and_execute {
-		let admin: T::AccountId = create_admin::<T>(0);
-		let n in 1..T::MaxTranches::get();
-		let tranches = build_update_tranches::<T>(n);
-		prepare_asset_registry::<T>();
-		create_pool::<T>(n, admin.clone())?;
-	}: update(RawOrigin::Signed(admin), POOL, PoolChanges {
-		tranches: Change::NewValue(build_update_tranches::<T>(n)),
-		min_epoch_time: Change::NewValue(SECS_PER_DAY),
-		max_nav_age: Change::NewValue(SECS_PER_HOUR),
-		tranche_metadata: Change::NewValue(build_update_tranche_metadata::<T>()),
-	})
-	verify {
-		// No redemption order was submitted and the MinUpdateDelay is 0 for benchmarks,
-		// so the update should have been executed immediately.
-		let pool = get_pool::<T>();
-		assert_update_tranches_match::<T>(pool.tranches.residual_top_slice(), &tranches);
-		assert_eq!(pool.parameters.min_epoch_time, SECS_PER_DAY);
-		assert_eq!(pool.parameters.max_nav_age, SECS_PER_HOUR);
-	}
+	// create {
```

Comment:
I think it would be nice to have a comment explaining why this section is here but commented out, but I'm fine with it as is.